components/network/mosquitto/pkg5

```diff
@@ -0,0 +1,14 @@
+{
+    "dependencies": [
+        "SUNWcs",
+        "library/security/openssl",
+        "shell/ksh93",
```

Comment:
You'll need to update this file. The dependencies seem to have changed from openssl to openssl-11

Reply:
Did change it later on, forgot to update. Fixed.